README.md

```diff
@@ -1,19 +1,36 @@
 [![Build Status](https://travis-ci.org/mirkobronzi/cookiecutter-pyml.svg?branch=master)](https://travis-ci.org/mirkobronzi/cookiecutter-pyml)
 
+About 
+-----
+
+A cookiecutter is a generic project template that will instantiate a new project with sane defaults. This repo contains our custom cookiecutter (`cookiecutter-pyml`) which will generate a new python deep learning package preconfigured with best practices in mind. It currently supports:
+
+* Pytorch/Tensorflow
+* Travis CI
+* Codecov
+* Sphinx (docs)
+* MLFlow (experiment management)
+* Orion (hyperparameter optimization)
+* Flake8
+* Pytest
+
+More information on what a cookiecutter is [here.](https://cookiecutter.readthedocs.io/en/)
+
 Quickstart
 ----------
 
```

Comment:
Do not clone this repository directly. This part will be handled by the cookiecutter command below.
Indeed, just follow the two commands below in order to generate your new deep learning project.